charts/generic-service/Chart.yaml

```diff
@@ -7,4 +7,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.0
+version: 1.5.1
```

Comment:
patch version since new functionality and will have no impact on existing services

Reply:
I suppose if you really want to be strict it is a new feature so should be 1.6.0 but not breaking- but not sure anyone cares

Reply:
agreed